Makefile

```diff
@@ -1102,10 +1074,10 @@ ocamldebugger: ocamlc ocamlyacc ocamllex otherlibraries
 partialclean::
 	$(MAKE) -C debugger clean
 
-# Check that the stack limit is reasonable.
-ifeq "$(UNIX_OR_WIN32)" "unix"
+# Check that the stack limit is reasonable (Unix-only)
 .PHONY: checkstack
 checkstack:
+ifeq "$(UNIX_OR_WIN32)" "unix"
```

Comment:
Your `Makefile`-fu is superior to mine - is this one of those cases where it would be a good idea for the Windows part to have:

```make
	@
```

(i.e. have the Windows target being a recipe which does nothing, rather than a target with no recipe)?

Reply:
I don't think it makes a difference in behavior, but I'll do it because it improves readability.